azure/Guidelines.md

```diff
@@ -846,109 +848,189 @@ https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#perfo
+## Pattern to Poll a Long-Running Operation's Status
 
-<a href="#lro-status-monitor-retry-after" name="lro-status-monitor-retry-after">:white_check_mark:</a> **DO** include a `Retry-After` header in the response to GET requests to the status monitor if the operation is not complete. The value of this header should be an integer number of seconds to wait before making the next request to the status monitor.
+<a href="#lro-poll" name="lro-poll">:white_check_mark:</a> **DO** use the following pattern to allow clients to poll the current state of a Status Monitor resource:<p>
+```
+GET /operations/<operation-id>
+```
 
-<a href="#lro-status-monitor-post-action-result" name="lro-status-monitor-post-action-result">:white_check_mark:</a> **DO** include the `result` property (if any) in the status monitor for a POST action-type long-running operation when the operation completes successfully.
+The response must look like this:<p>
+```
+200 OK
+retry-after: <delay-seconds>    (if status not terminal)
+<JSON Status Monitor Resource in body>
+```
 
-<a href="#lro-status-monitor-no-resource-result" name="lro-status-monitor-no-resource-result">:no_entry:</a> **DO NOT** include a `result` property in the status monitor for a long-running operation that is not a POST action-type long-running operation.
+<a href="#lro-poll-accepts-any-api-version" name="lro-poll-accepts-any-api-version">:ballot_box_with_check:</a> **YOU SHOULD** allow any valid value of the `api-version` query parameter to be used in the GET operation on the status monitor.
+
+  - Note: Clients may replace the value of `api-version` in the `operation-location` URL with a value appropriate for their application. Remember that the client initiating the LRO may not be the same client polling the LRO's status.
 
 <a href="#lro-status-monitor-retention" name="lro-status-monitor-retention">:white_check_mark:</a> **DO** retain the status monitor resource for some publicly documented period of time (at least 24 hours) after the operation completes.
 
+## Pattern to List Status Monitors (optional)
+
+<a href="#lro-list" name="lro-list">:white_check_mark:</a> **DO** use the following pattern to allow clients to list Status Monitor resources:<p>
+```
+GET /operations?kind=VMInitializing,VMRebooting&status=NotStarted,Succeeded
+```
+
+The response must look like this:<p>
+```200 OK
+{
+    "value": [
+       { "id": "12345", "kind": "VMInitializing", "status": "Running", … },
+       { … },
+       { … },
+       { "id": "abcde", "kind": "VMRebooting", "status": "Failed", … }
+    ],
+    "nextLink": "{opaqueUrl}"
+}
+```
+ - The values for the `kind` and `status` query parameters are logically OR'd together. 
+ 
+   - The above example returns all status monitor resources whose `kind` is either "VMInitializing" *or* "VMRebooting" and whose status is "NotStarted" *or* "Succeeded".
```

Comment:
```suggestion
- The above example returns all status monitor resources whose `kind` is either "VMInitializing" *or* "VMRebooting" and whose status is "NotStarted" *or* "Succeeded".
```

Please check all bullets. I've seen a number misaligned and it does matter. If you're using VSCode, you can click the Preview button. I also recommend installing the Microsoft Learn Pack that includes markdownlint et. al. It's what's required for docs on learn.microsoft.com (validation will run the same rules so it's best to install it).

Reply:
I am using markdown lint, but I'll double-check the rendered version after pushing this round of fixes.